packages/lavapack/src/runtime.js

```diff
@@ -4408,27 +4420,64 @@ const loadWithoutErrorAnnotation=  async(
       } in compartment ${q(compartment.name)}`;
    }
 
-  if( staticModuleRecord.record!==  undefined) {
```

Comment:
The changes in this file seem unrelated to the CJS generation update of this PR? Can it be broken out as a separate change?